test/test_bot.py

```diff
@@ -368,32 +368,32 @@ def nick_command_info(bot, trigger):
     def action_command_tell(bot, trigger):
         pass
 
+    @module.commands('mixed')
+    @module.rule('mixing')
+    def mixed_rule_command(bot, trigger):
+        pass
+
     @module.event('JOIN')
     @module.label('handle_join_event')
     def on_join(bot, trigger):
         pass
 
     # prepare callables to be registered
-    loader.clean_callable(rule_hello, tmpconfig)
-    loader.clean_callable(command_do, tmpconfig)
-    loader.clean_callable(command_main_sub, tmpconfig)
-    loader.clean_callable(command_main_other, tmpconfig)
-    loader.clean_callable(nick_command_info, tmpconfig)
-    loader.clean_callable(action_command_tell, tmpconfig)
-    loader.clean_callable(on_join, tmpconfig)
-
     callables = [
         rule_hello,
         command_do,
         command_main_sub,
         command_main_other,
         nick_command_info,
         action_command_tell,
+        mixed_rule_command,
         on_join,
     ]
 
-    # set plugin name by hand, because plugins are supposed to do it
+    # clean callables and set plugin name by hand
+    # since the plugin's handler is supposed to do it
```

Comment:
```suggestion
    # since the loader and plugin handlers are excluded here
```

Just addressing both steps in the same comment, is all.